src/PropertiesUtils.cpp

```diff
@@ -37,6 +37,16 @@ std::string RightTrim(const std::string& str) {
 }
 
 std::string LeftTrim(const std::string& str) {
+  std::string rstr = ltrim(str);
+
+  while (rstr != ltrim(rstr)) {
+    rstr = ltrim(rstr);
+  }
+
+  return rstr;
+}
+
+std::string ltrim(const std::string& str) {
```

Comment:
Can you move `ltrim` to the anonymous `namespace` in the same file? That way, it can only be used this file.